### PR TITLE
Add a caret icon to test suites folder

### PR DESCRIPTION
**Pull request by callingmedic911**
Closes SCS-1666.

Adds an caret icon to right of folder name to make clear that section is collapsible. https://www.loom.com/share/7ef89c3b5fc34722b98d2e280e68b4a6

@jonbell-lot23 can you please check this looks right before I pass review to frontend team?

### CODE REVIEW

**Comment by jonbell-lot23** [comment]
Thanks, @callingmedic911 ! Let's try putting the carets directly next to the name, rather than aligned on the far right:

directory / directory / name <

Rather than 

directory / directory / name                                   <

**Comment by bvaughn** [comment]
The caret alignment (right side alignment) is off
Screenshot 2023-12-11 at 9 44 50 AM

File: src/ui/components/Library/Team/View/NewTestRuns/Overview/RunResults.tsx [hunk 1]
```diff
@@ -206,20 +206,29 @@ function PathNodeRenderer({
     <>
       {name && (
         <div
-          className={`cursor-pointer truncate rounded py-2 pr-4 ${styles.libraryRow} ${
-            containsSelectedSpec ? styles.libraryRowSelected : ""
-          }`}
+          className={`flex cursor-pointer justify-between truncate rounded py-2 pr-4 ${
```
Comment (bvaughn) [inline src/ui/components/Library/Team/View/NewTestRuns/Overview/RunResults.tsx:209]: I think you can fix the right alignment by replacing "pr-4" with "pr-2"

Comment (bvaughn) [reply src/ui/components/Library/Team/View/NewTestRuns/Overview/RunResults.tsx:209]: The padding is too much.

Comment (callingmedic911) [reply src/ui/components/Library/Team/View/NewTestRuns/Overview/RunResults.tsx:209]: I see what you mean. I'll be addressing Jon's comment above, so it won't be aligned to right anyway. I'll remove right padding.

**Comment by callingmedic911** [comment]
Icon is now next to path names. https://www.loom.com/share/746e5735bf4344559ff67262de1fd6d2

**Review by bvaughn — APPROVED** [review]

**Pull request merged**